torch/distributed/distributed_c10d.py

```diff
@@ -117,11 +120,11 @@ def register_backend(cls, name, func):
 # `_backend`, `dist_backend`, and `reduce_op` are here to maintain backward
 # compatibility with pre-c10d distributed package.
 # TODO: remove them when users are ready to take a hard dependency on PyTorch 1.
-_backend = Backend.UNDEFINED
+_backend: str = Backend.UNDEFINED
 dist_backend = Backend
 
 
-class reduce_op(object):
+class _reduce_op(object):
```

Comment:
Need to rename this class because reduce_op is later assigned to an instance of this class.